documentation/prerequisites.md

```diff
@@ -7,13 +7,15 @@ Following is the list of prerequisites for running Kubernetes with vSphere Cloud
 * We recommend Kubernetes version 1.6.5 and vSphere version 6.0.x
 * We support Photon, Ubuntu, CoreOs and Centos. it has been tested on Photon 1.0 GA, Ubuntu 16.04, CoreOS 4.11.2 and Centos 7.3
 * vSphere setup to deploy the Kubernetes cluster.
+   - For kubernetes version 1.9.x and above: vSphere Cloud Provider supports kubernetes cluster spnning across multiple vCenters.
+   - For kubernetes version 1.8.x and below: vSphere Cloud Provider supports kubernetes cluster deployed only in one vCenter.
 * vSphere shared storage. It can be any one of VSAN, sharedVMFS, sharedNFS.
    - Shared storage allows all the nodes in the Kubernetes cluster to access the storage blocks.
```

Comment:
Let's remove this line. We should not be explaining the rationale behind each prerequisite.

Reply:
Done